xExtension-ImageProxy/extension.php

```diff
@@ -3,7 +3,7 @@
 class ImageProxyExtension extends Minz_Extension {
 	public function init() {
 		$this->registerHook('entry_before_display',
-		                    array('ImageProxyExtension', 'setImageProxyHook'));
+                            array('ImageProxyExtension', 'setImageProxyHook'));
```

Comment:
It should either be one tab or however many spaces it was. ;-)

Reply:
PS Note that's two tabs followed by alignment spaces.

Reply:
lol noticed that, trying to copy-paste this correctly from the initial code (that's what I thought I'd done, but looks I missed stg :D)